src/content/structured/community/index.mdx

```diff
@@ -19,3 +19,41 @@ We hope to demonstrate our approach to software and inclusive design.
 - See our [roadmap](/community/roadmap) for an overview of recent work, current focus and work we have planned.
 - Learn how you can [contribute](/community/contribute) using the [contribution criteria](/community/contribute-criteria).
 - How our community approaches design through our [design principles](/get-started/design-principles).
+
+### More open source from the Intelligence Community
```

Comment:
```suggestion
## More open source from the Intelligence Community
```

Reply:
Not sure which formatting we want here so this is just a suggestion.

Reply:
I agree that heading should be on the same level as the introduction :)  Have made the change